### DISCUSSION TITLE
Would this work for grammar?

### DISCUSSION THREAD

**Discussion by Kaszebe** [brucepro/Memoir#72] [Q&A]
If I wanted the LLM to adhere to a very strict set of grammar rules (e.g. "don't use passive voice" or "don't write complex sentences," could this extension help?

**Comment by brucepro** [comment]
Usually the system prompt is the best place for those kind of instructions. Memoir is more of a memory tool, so the A.I. would remember the various times you have discussed not using passive voice or don't use complex sentences.  Just ask your A.I. to generate a statement to add to the system prompt to use an active voice and write simple sentences.  Then add that to the system prompt for your persona.
